tests/test_cli.py

```diff
@@ -313,7 +314,8 @@ def test_schema_multi_expand(self):
                     'format': {'enum': ['csv', 'json', 'txt', 'csv2dict']},
                     'expr': {'oneOf': [
                         {'type': 'integer'},
-                        {'type': 'string'}]}
+                        {'type': 'string'}]},
+                    'headers': {'type': 'object'},
```

Comment:
ideally use pattern properties in jsonschema to restrict the values to strings

Reply:
It is also updated with `patternProperties` definition.